WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -102,11 +101,11 @@ protected SendControlViewModel(Wallet wallet, string title)
 				{
 					if (Money.TryParse(x.TrimStart('~', ' '), out Money amountBtc))
 					{
-						SetAmountWatermark(amountBtc);
+						AmountTip = amountBtc.ToUsdString(Wallet.Synchronizer.UsdExchangeRate, false);
```

Comment:
LWM mode should work. What was the problem there?

Reply:
I don't think it makes sense to have LWM for the tooltip while the price is visible.

Reply:
It should work the same as it works when you hover over your mouse on your total balance in wallet explorer. LWM mode is working there too and `ToUsdString` requires the LWM parameter. Something wrong there, let's figure out what - do not set LWM to false.

Reply:
I revoke my last comment. The amount is shown there, even in LWM mode, the price is visible too - so the tooltip can be visible too.

Reply:
Anyway just to make sure we don't have a bug can you elaborate why it is showing ### when the amount was zero?

Reply:
You mean in your picture? Because the picture was before https://github.com/zkSNACKs/WalletWasabi/pull/4060/commits/36e769e9ae92d9ebbd1675bd676f8e6c713e932f